spec/ddtrace/contrib/mysql2/patcher_spec.rb

```diff
@@ -10,24 +10,23 @@
     Mysql2::Client.new(
       host: host,
       port: port,
-      database: database
+      database: database,
+      password: 'root'
```

Comment:
I think you might also need to swap this variable for a configured one, just like you did for `database`. In this case, it should be `TEST_MYSQL_ROOT_PASSWORD` with the default value of `root`.